src/bundle/Resources/public/js/alloyeditor/src/plugins/ez-custom-tag.js

```diff
@@ -263,6 +265,8 @@
                 setAlignment: function(type) {
                     this.wrapper.data(DATA_ALIGNMENT_ATTR, type);
                     this.element.data(DATA_ALIGNMENT_ATTR, type);
+                    window.clearTimeout(this.setAlignmentFireEditorInteractionTimeout);
+                    this.setAlignmentFireEditorInteractionTimeout = window.setTimeout(this.fireEditorInteraction.bind(this,'aligmentUpdated'), 50);
```

Comment:
Missing space after `,` in `bind(this,'aligmentUpdated')`, applies in other cases.